### ISSUE TITLE
Resources are being doubled on active vessel when quicksaving and reverting flights on 1.8

### ISSUE THREAD

**Issue by aidygus** [allista/ConfigurableContainers#32]
I've noticed an issue where resources are being doubled on all tanks on the active vessel when quick saving games or reverting a flight to launch.  I noticed the problem as I use KCT to reuse vessels such as shuttles and SSTOs and on the 4th flight the FPS would go into the yellow then on the 5th would crash KSP when launching craft due to each tank having sometimes hundreds of resources assigned per tank due to quicksaving during flight then being recovered and stored by KCT.  My space station also started suffering from FPS drops as I quick save there a lot.  Manually editing the .sfs file to remove the excess tanks returned FPS to normal but I now have to regularly prune the resources from the tanks to keep it up as I do save a lot at the station.

**Comment by allista** [comment]
Thanks for the report. This is a known issue which unfortunately was discussed only in private messages on the forum.

What we found out is that it's the bug in Remote Tech that causes the duplication, so there's nothing I can do.

Here's the issue in RT tracker that explains the problem: 
https://github.com/RemoteTechnologiesGroup/RemoteTech/issues/805

**Comment by aidygus** [comment]
That explains one of my polar satellites.  It had 4096 defined resources across 3 tanks and added over 10 Mb to the save file size when I first started pruning.  It's in a similar altitude to my space station so every time it loaded when in range the tanks were duplicating.  I was wondering how it had so many as after launching it and collecting science I haven't flown it again.  Lets hope RT fix the issue soon

**Comment by KSP-TaxiService** [comment]
Hi @allista,

Just to be sure I follow the reproduction steps correctly.

Reproduction steps:
1. Launch a vessel with initial resource amounts
2. Press F5 (Quicksave)
3. Open the "Manage tanks" dialog to confirm the amounts are unchanged (visual only)
4. Press F9 (Quickload)
5. Open the dialog again to observe the amounts are increased by 2 factors, like this picture below
![2](https://user-images.githubusercontent.com/22871614/70334227-019a6e80-1880-11ea-96ce-26ef0ee63233.PNG)

It seems that this issue (doubling resource amounts and excessive save size) only happens when RemoteTech and Configurable Containers are installed together. Is it correct?

Thanks.

**Comment by allista** [comment]
@KSP-TaxiService correct; except that not the amount of resources increases, but the tanks themselves are duplicated.

**Comment by KSP-TaxiService** [comment]
Hi @allista,

I eliminated the relevant codes and verified that it does not generate duplicate configuration values in my tests.
Kindly assist to verify the fixed DLL at your side.
[RemoteTech_DLL.zip](https://github.com/allista/ConfigurableContainers/files/3945242/RemoteTech_DLL.zip)

I believe this issue was causing the few other issues I encountered outside "Configurable Containers" mod.

Thanks.
